### PR TITLE
  Add recipes for docker manifests and tags copy and remove operations.

### PR DESCRIPTION
**Pull request by ipanova**

### CODE REVIEW

File: docs/user-guide/recipes.rst [hunk 1]
```diff
@@ -489,3 +482,130 @@ command::
...
+
+    [|]
+    Running...
+
+    Copied:
+      docker_blob: 93
+      docker_manifest: 115
+
+
+As you can see during the copy of image manifests all referenced blobs where carried over as well.
```
Comment (mhrivnak) [inline docs/user-guide/recipes.rst]: Add a comma after "manifests".

Comment (mhrivnak) [reply docs/user-guide/recipes.rst]: s/where/were/

File: docs/user-guide/recipes.rst [hunk 2]
```diff
@@ -489,3 +482,130 @@ command::
...
+    Description: content copy commands
+
+    Available Commands:
+      image         - copies images from one repository into another
+      manifest      - copies manifests from one repository into another
+      manifest-list - copies manifest lists from one repository into another
+      tag           - copies tags from one repository into another
+
+
+* If a manifest list is copied - all listed image manifests within the manifest list and blobs
```
Comment (mhrivnak) [inline docs/user-guide/recipes.rst]: s/ -/,/

File: docs/user-guide/recipes.rst [hunk 3]
```diff
@@ -489,3 +482,130 @@ command::
...
+  will be carried over. Tags of images manifests will not be copied.
+* If a tag which references an image manifest is coped - image manifest and all its blobs will
```
[the remark this answers is not in the document]
Comment (mhrivnak) [reply docs/user-guide/recipes.rst]: s/ -/,/

File: docs/user-guide/recipes.rst [hunk 4]
```diff
@@ -489,3 +482,130 @@ command::
...
+  be copied over.
+* If a tag which references a manifest list is coped - manifests list, all listed image manifests
```
Comment (mhrivnak) [inline docs/user-guide/recipes.rst]: s/ - /, the/

File: docs/user-guide/recipes.rst [hunk 5]
```diff
@@ -489,3 +482,130 @@ command::
...
+    Usage: pulp-admin [SUB_SECTION, ..] COMMAND
+    Description: content removal commands
+
+    Available Commands:
+      image         - remove images from a repository
+      manifest      - remove manifests from a repository
+      manifest-list - remove manifest lists from a repository
+      tag           - remove tags from a repository
+
+* If a tag is removed - just the tag itself will be removed from the repository.
```
Comment (mhrivnak) [inline docs/user-guide/recipes.rst]: s/ -/,/

File: docs/user-guide/recipes.rst [hunk 6]
```diff
@@ -489,3 +482,130 @@ command::
...
+* If a manifest list is removed - all its image manifests which don't have tags and are not
```
Comment (mhrivnak) [inline docs/user-guide/recipes.rst]: s/ -/,/

**Comment by ipanova** [comment]
@mhrivnak i removed the 2.5 release notes because it got never released, instead the changes got released in 2.4.1

**Comment by ipanova** [comment]
@mhrivnak back to you, i should have probably target it against 3.0-dev branch

File: docs/user-guide/recipes.rst [hunk 7]
```diff
@@ -489,3 +482,130 @@ command::
...
+
+    Available Commands:
+      image         - copies images from one repository into another
+      manifest      - copies manifests from one repository into another
+      manifest-list - copies manifest lists from one repository into another
+      tag           - copies tags from one repository into another
+
+
+* If a manifest list is copied, all listed image manifests within the manifest list and blobs
+  will be carried over. Tags of images manifests will not be copied.
```
Comment (mhrivnak) [inline docs/user-guide/recipes.rst]: s/images/image/

File: docs/user-guide/recipes.rst [hunk 8]
```diff
@@ -489,3 +482,130 @@ command::
...
+* If a tag which references an image manifest is copied, image manifest and all its blobs will
+  be copied over.
+* If a tag which references a manifest list is copied, the manifests list, all listed image manifests
```
Comment (mhrivnak) [inline docs/user-guide/recipes.rst]: s/manifests/manifest/

**Pull request merged**